Print_A_Bot/Print_A_Bot/settings.py

```diff
@@ -38,35 +39,12 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
 
+    'channels',
+    # 'rest_framework',
+
     # Apps
     'controls',
-    #'rest_framework',
-
-    ]
-
-
-# Raspberry Pi Config
```

Comment:
I moved these to `config.py` in the root directory, they're likely to return 

original goal was to have a minimal process that could run independent of django but with channels it doesn't make so much sense (I blame the codeine)